js/src/figure.js

```diff
@@ -142,7 +150,8 @@ var FigureView = widgets.DOMWidgetView.extend( {
         this.wire_box.add(make_line(-0.5, -0.5+1, -0.5, -0.5, -0.5+1, -0.5+1, this.axes_material))
         this.wire_box.add(make_line(-0.5+1, -0.5+1, -0.5, -0.5+1, -0.5+1, -0.5+1, this.axes_material))
 
-        this.camera.position.z = 2
+        // set a good intial z for any fov angle
+        this.camera.position.z = 2 * this.getTanDeg(45/2) / this.getTanDeg(this.model.get("camera_fov")/2)
```

Comment:
Just some 'back of an envelope' trigonometry: 

![fullsizerender](https://user-images.githubusercontent.com/2997570/28863399-a015c41a-7760-11e7-9ebc-55bbdcae163f.jpg)

Since the camera is intialized at (0,0,z) (i.e. a=z) and we know that theta=45 and z=2 provides a good initial view (as is currently hard coded), hence this line.

Make sense?